src/qibocal/protocols/flipping.py

```diff
@@ -23,23 +23,38 @@
 
 
 def flipping_sequence(
-    platform: CalibrationPlatform, qubit: QubitId, delta_amplitude: float, flips: int
+    platform: CalibrationPlatform,
+    qubit: QubitId,
+    delta_amplitude: float,
+    flips: int,
+    rx90: bool,
 ):
     """Pulse sequence for flipping experiment."""
 
     sequence = PulseSequence()
     natives = platform.natives.single_qubit[qubit]
-    sequence |= natives.R(theta=np.pi / 2)
+
+    if rx90:
+        sequence |= natives.RX90()
+    else:
+        sequence |= natives.R(theta=np.pi / 2)
 
     for _ in range(flips):
 
-        qd_channel, rx_pulse = natives.RX()[0]
+        if rx90:
+            qd_channel, qd_pulse = natives.RX90()[0]
+        else:
+            qd_channel, qd_pulse = natives.RX()[0]
 
-        rx_detuned = update.replace(
-            rx_pulse, amplitude=rx_pulse.amplitude + delta_amplitude
+        qd_detuned = update.replace(
+            qd_pulse, delta_amplitude=qd_pulse.amplitude + delta_amplitude
```

Comment:
```suggestion
            qd_pulse, amplitude=qd_pulse.amplitude + delta_amplitude
```
Shouldn't this be `amplitude`?